script.js

```diff
@@ -0,0 +1,164 @@
+// 1
+
+const teamRegistration = (
+  teamName,
+  driverName,
+  yearOfbirth,
+  carBrand,
+  sponsor,
+  allowedTorally
+) => {
+  teamName = prompt("Enter name of your team");
+  driverName = prompt("Enter your name");
+  yearOfbirth = +prompt("Enter your year of birth");
+  carBrand = prompt("Enter your car brand");
+  sponsor = prompt("Enter name of your sponsor");
+
+  if (
+    teamName === null ||
+    driverName === null ||
+    yearOfbirth === null ||
+    carBrand === null ||
+    sponsor === null
+  ) {
+    allowedTorally = false;
+  } else if (
+    teamName === "" ||
+    driverName === "" ||
+    yearOfbirth === "" ||
+    carBrand === "" ||
+    sponsor === ""
+  ) {
+    allowedTorally = false;
+  } 
+  else if (
+    isNaN(yearOfbirth) 
+  ) {
+    allowedTorally = false;
+  } 
+  else {
+    allowedTorally = true;
+  }
+  return {
+    name: driverName,
+    yearOfbirth: yearOfbirth,
+    car: carBrand,
+    sponsor: sponsor,
+    allowedTorally: allowedTorally,
+  };
+};
+const team_1 = teamRegistration();
+console.log(team_1);
+
+//2
+
+const userRegistration = () => {
+  const newUser = {
+    login: "",
+    password: "",
+    city: "",
+    country: "",
+    gender: "",
+    age: "",
+  };
+  for (let key in newUser) {
+    let userInfo = prompt(`Enter ${key}`);
+    if (userInfo === null || userInfo === "") {
+      console.log("Enter any value");
+    } else {
+      newUser[key] = userInfo;
+    }
+  }
+  return newUser;
+};
+const newUser = userRegistration();
+
+//3
+
+const updateUserData = (object, key, value) => {
+  return (object[key] = value);
+};
+
+updateUserData(userRegistration, "city", "Kherson");
+
+//4
+
+const studentsObject = {
+  name: "Adam",
+  surname: "Sandler",
+  age: 25,
+  course: 5,
+  city: "Los Angeles",
+  greeting: function () {
+    console.log("Hi. everyone!");
+  },
+  addHomework: function () {
+    console.log("Sending my howework... Please, wait");
+  },
+};
+studentsObject.greeting();
+studentsObject.addHomework();
+
+//5
+
+const isEmpty = (newUser) => {
+  for (let key in newUser) {
+    console.log(false);
+    return false;
+  }
+  console.log(true);
+  return true;
+};
```

Comment:
It can be simplified 
```suggestion
  return Object.keys(value).length === 0 ? true : false
```

Or even `return Object.keys(value).length === 0`